alloy/src/main/resources/META-INF/resources/liferay-faces-alloy/alloy.js

```diff
@@ -598,7 +598,91 @@ LFAI = {
 		});
 	},
 
+	initTabView: function(tabView, selectedIndexClientId, clientId, tabSelectClientBehaviors) {
+
+		tabView.after('selectionChange', function(event) {
+
+			var selectedIndexInput = document.getElementById(selectedIndexClientId);
+
+			if (event.newVal) {
+
+				selectedIndexInput.value = LFAI.getServerSideIndex(event.newVal.get('srcNode').get('parentNode'));
+
+				for (var i = 0; i < tabSelectClientBehaviors.length; i++) {
+					tabSelectClientBehaviors[i]();
+				}
+			}
+			else {
+				selectedIndexInput.value='';
+			}
+		});
+
+		// The outermost div was initially styled with "display:none;" in order to prevent blinking when Alloy's
+		// JavaScript attempts to hide the div. At this point in JavaScript execution, Alloy is done manipulating the
+		// DOM and it is necessary to reset the style so that the component will be visible.
+		document.getElementById(clientId).style.display = null;
+	},
+
+	initAccordion: function(multiple, accordion, selectedIndexClientId, clientId, tabExpandedClientBehaviors,
```

Comment:
@stiemannkj1 Functions that we add to alloy.js are technically API. If that is the case, then is there a convention we should use to indicate that a method is not API, such as starting a function name with an underscore?